docker/calamari.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:20.04 as builder
+LABEL description="run calamari binary distribution in docker"
+ARG TAG_NAME
+ARG BINARY="https://github.com/Manta-Network/Manta/releases/download/${TAG_NAME}/calamari-pc"
+ARG CALAMARI_GENESIS="https://raw.githubusercontent.com/Manta-Network/Manta/manta-pc/genesis/calamari-genesis.json"
```

Comment:
i believe that we should not be using mutable urls such as this. this url allows for change to the genesis chainspec since it points to the tip of the `manta-pc` branch, which can result in multiple different instantiations of the same docker image or a failed instantiation, if someone removes the chainspec from the `manta-pc` branch.
ideally, we ought to use the same tag as that used to reference the calamari-pc binary, so `https://raw.githubusercontent.com/Manta-Network/Manta/${TAG_NAME}/genesis/calamari-genesis.json` would be the right way to reference the chainspec.
if that isn't possible (because the tag does not contain the genesis chainspec), then a specific git sha should be used to identify the exact version of the chainspec that the docker image should contain.